move_together_app/lib/core/models/message.dart

```diff
@@ -30,6 +35,11 @@ class Message {
       'createdAt': createdAt.toIso8601String()
     };
   }
+
+  bool isCurrentUserAuthor(BuildContext context) {
+    final userId = context.read<AuthProvider>().userId;
+    return author.id == userId;
+  }
```

Comment:
```suggestion
  bool isCurrentUserAuthor(BuildContext context) {
    return author.isMe(context);
  }
```

Tu peux pas faire ca comme ca ? Comme ca été fait pour les trips avec les participants ?